docs/guides/update-guide/820-to-830.md

```diff
@@ -0,0 +1,28 @@
+---
+id: 820-to-830
+title: Update 8.2 to 8.3
+description: "Review which adjustments must be made to migrate from Camunda Platform 8.2.x to Camunda Platform 8.3.0."
+---
+
+<span class="badge badge--primary">Intermediate</span>
+
+The following sections explain which adjustments must be made to migrate from Camunda Platform 8.2.x to 8.3.0 for each component.
+
+## Zeebe
+
+### Change in monitoring endpoints
+
+There were some changes to the monitoring endpoints of Zeebe, both for the gateway and the broker. All of these were existing redirects to the new endpoints, so their format is left unchanged; only the redirects have been removed.
+
+You may have to change your Kubernetes health probes or metrics scraping endpoints to accomodate the new ones. Note that if you're deploying with the Helm charts, or are using Camunda Cloud, you do not need to do anything.
```

Comment:
Do you mean Camunda Platform 8 here? Or pre 8.0 versions of the platform (aka Camunda Cloud)?

Reply:
My guess is you want to say "Camunda Platform 8 SaaS" here.

Reply:
+1